modules/serverless-common/pages/service-mesh/common-service-mesh-network-isolation.adoc

```diff
@@ -0,0 +1,438 @@
+        sidecar.istio.io/inject: 'true'
+        autoscaling.knative.dev/target-burst-capacity: "-1"
+    spec:
+      containers:
+        - image: docker.io/openshift/hello-openshift
+          env:
+            - name: RESPONSE
+              value: "Hello Serverless!"
+EOF
+
+cat <<-EOF | oc apply -f -
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: curl
+  namespace: tenant-1
+  labels:
+    app: curl
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: curl
+  template:
+    metadata:
+      labels:
+        app: curl
+      annotations:
+        sidecar.istio.io/inject: 'true'
+    spec:
+      containers:
+      - name: curl
+        image: curlimages/curl
+        command:
+        - sleep
+        - "3600"
+EOF
+
+# Tenant 2
+cat <<-EOF | oc apply -f -
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: test-webapp
+  namespace: tenant-2
+  annotations:
+    serving.knative.openshift.io/enablePassthrough: "true"
+spec:
+  template:
+    metadata:
+      annotations:
+        sidecar.istio.io/inject: 'true'
+        autoscaling.knative.dev/target-burst-capacity: "-1"
+    spec:
+      containers:
+        - image: docker.io/openshift/hello-openshift
+          env:
+            - name: RESPONSE
+              value: "Hello Serverless!"
+EOF
+----
+
+. Verification
++
+[source,terminal]
+----
+# Test tenant-1 -> tenant-1 via cluster local domain (allowed)
+oc exec deployment/curl -n tenant-1 -it -- curl -v http://test-webapp.tenant-1:80
+
+HTTP/1.1 200 OK
+content-length: 18
+content-type: text/plain; charset=utf-8
+date: Wed, 26 Jul 2023 12:49:59 GMT
+server: envoy
+x-envoy-upstream-service-time: 9
+
+Hello Serverless!
+
+
+# Test tenant-1 -> tenant-1 via external domain (allowed)
+EXTERNAL_URL=$(oc get ksvc -n tenant-1 test-webapp -o custom-columns=:.status.url --no-headers)
+oc exec deployment/curl -n tenant-1 -it -- curl -ik $EXTERNAL_URL
+
+HTTP/2 200
+content-length: 18
+content-type: text/plain; charset=utf-8
+date: Wed, 26 Jul 2023 12:55:30 GMT
+server: istio-envoy
+x-envoy-upstream-service-time: 3629
+
+Hello Serverless!
+
+
+# Test tenant-1 -> tenant-2 via cluster local domain (not allowed)
+oc exec deployment/curl -n tenant-1 -it -- curl -v http://test-webapp.tenant-2:80
+
+* processing: http://test-webapp.tenant-2:80
+*   Trying 172.30.73.216:80...
+* Connected to test-webapp.tenant-2 (172.30.73.216) port 80
+> GET / HTTP/1.1
+> Host: test-webapp.tenant-2
+> User-Agent: curl/8.2.0
+> Accept: */*
+>
+< HTTP/1.1 403 Forbidden
+< content-length: 19
+< content-type: text/plain
+< date: Wed, 26 Jul 2023 12:55:49 GMT
+< server: envoy
+< x-envoy-upstream-service-time: 6
+<
+* Connection #0 to host test-webapp.tenant-2 left intact
+RBAC: access denied
+
+
+# Test tenant-1 -> tenant-2 via external domain (allowed)
+EXTERNAL_URL=$(oc get ksvc -n tenant-2 test-webapp -o custom-columns=:.status.url --no-headers)
+oc exec deployment/curl -n tenant-1 -it -- curl -ik $EXTERNAL_URL
```

Comment:
same as above

Reply:
I would prefer to discuss the use kn as a separate topic. If we do this, we should do it over all the docs.